dnf/cli/commands/repoquery.py

```diff
@@ -111,12 +112,15 @@ def fmt_repl(matchobj):
     return fmt
 
 
-@dnf.plugin.register_command
-class RepoQueryCommand(dnf.cli.Command):
+class RepoQueryCommand(commands.Command):
+    """A class containing s needed by the cli to execute the
+    reinstall command.
+    """
 
-    """The util command there is extending the dnf command line."""
+    activate_sack = True
```

Comment:
This does not work with new argparser changes it has to be set via demands in configure().